cmd2/argparse_completer.py

```diff
@@ -510,6 +510,18 @@ def complete_command_help(self, tokens: List[str], text: str, line: str, begidx:
                             return self.basic_complete(text, line, begidx, endidx, completers.keys())
         return []
 
+    def format_help(self, tokens: List[str]) -> str:
```

Comment:
This looks OK to me.  Consider updating the CHANGELOG and/or adding a unit test for this method.